desc/plotting.py

```diff
@@ -320,7 +320,9 @@ def _compute(eq, name, grid, component=None, reshape=True):
     return data, label
 
 
-def _compute_Bn(eq, field, plot_grid, field_grid):
+def _compute_Bn(
+    eq, field, plot_grid, field_grid, chunk_size=None, B_plasma_chunk_size=None
```

Comment:
do we need to add a pass through kwarg to `plot_xx` so these can actually be set by the calling function?